apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/service/NamespaceService.java

```diff
@@ -256,20 +256,24 @@ private NamespaceBO transformNamespace2BO(Env env, NamespaceDTO namespace) {
 
   private void fillAppNamespaceProperties(NamespaceBO namespace) {
 
-    NamespaceDTO namespaceDTO = namespace.getBaseInfo();
+    final NamespaceDTO namespaceDTO = namespace.getBaseInfo();
```

Comment:
what is the usage of `final` here?

Reply:
Just a trick to prevent me from writing wrong code. (Like wrong assign operation).